src/domain/rules/entities/SynchronizationRule.ts

```diff
@@ -535,8 +545,8 @@ export class SynchronizationRule {
         return this.update({ frequency });
     }
 
-    public updateLastExecuted(lastExecuted: Date): SynchronizationRule {
-        return this.update({ lastExecuted });
+    public updateLastExecuted(lastExecuted: Date, id: string, name: string): SynchronizationRule {
```

Comment:
Instead of ``id: string, name: string`` we probably want ``lastUpdatedBy: NamedRef``